_posts/2025-09-08-open-horizons.md

```diff
@@ -0,0 +1,343 @@
+
+2. **Nurture a Growth Mindset**
+   Embrace learning and development as ongoing processes. Recognize strengths and weaknesses without judgment and see challenges as opportunities to grow rather than threats.
+3. **Planning Over Plans**
+   Prioritize adaptability and momentum over rigid adherence to plans. Use planning as a tool to guide your actions, not as a constraint that stifles responsiveness or creativity.
+4. **Sustain Momentum**
+   Sustain momentum by regularly reflecting, celebrating wins, and making thoughtful adjustments. Build margin to absorb shocks and enable bold swings, ensuring your growth remains dynamic, resilient, and adaptable.
+5. **Leverage Strengths**
+   Build on what energizes and empowers you. While growth areas are important, your strengths are your foundation for success and resilience.
+
+## Nested Feedback Loops
+
+A healthy growth system relies on nested feedback loops operating at different scales. Small-scale rituals, like daily focus or weekly reviews, strengthen and inform larger ones such as quarterly reflections and mission affirmations. Conversely, these larger-scale rituals guide and protect the smaller ones, creating a dynamic balance that sustains growth.
+
+Don’t drop a ritual without substituting something that preserves the same signal at the same scale. If a practice is removed, it’s essential to explicitly replace it with something that provides feedback at the same scale and frequency to maintain this balance. Think of it as "defense in depth"—if one ritual misses something, another catches it, ensuring nothing slips through the cracks. This layered approach to feedback keeps your growth system resilient and responsive.
+
+## Activities Overview
+
+| Activity                | Purpose                                    | Frequency       | Principle Emphasized      |
+|-------------------------|--------------------------------------------|-----------------|--------------------------|
+| Reflect on Achievements | Ground direction in real impact            | Quarterly       | Aim With Clarity          |
+| Recognize Strengths     | Identify and leverage personal strengths   | Quarterly       | Nurture a Growth Mindset |
+| Affirm Your Mission     | Clarify motivating purpose                  | Quarterly       | Aim With Clarity          |
+| Aim & Set Priorities    | Define focused growth areas                  | Quarterly       | Planning Over Plans       |
+| Weekly Review           | Break priorities into actionable tasks     | Weekly          | Planning Over Plans       |
+| Daily Focus             | Connect daily work to long-term aims       | Daily           | Sustain Momentum          |
+
+Starting Open Horizons may feel like a significant commitment. The initial run-through of these activities usually takes 4–6 weeks as you build habits and understanding. A practical approach is to focus on one activity per week, allowing each ritual to embed naturally before adding the next.
```

Comment:
> A practical approach is to focus on one activity per week, allowing each ritual to embed naturally before adding the next.

I think a reader could misinterpret this to mean that they should _always_ focus on one activity per week. But I believe instead that you're suggesting this as an onboarding strategy.

Perhaps something like this would avoid this potential misinterpretation?

> When getting started, a practical approach is to adopt one activity per week, allowing each ritual to embed naturally before adding the next.